### PR TITLE
Fix/5.3/update info version

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1159,6 +1159,8 @@ Posting a News in the activity stream
 It is possible to write an article directly from a space composer. This feature is intended to improve and simplify the way information is shared. Users can contribute content intuitively without any former training to the content management system. News offers an inclusive and collaborative way to manage communication.
 Publishing a news lets you easily write, broadcast  and share communication content into other spaces. You can also make it visible to all platform users by pinning articles on the home page.
 
+.. warning:: News feature is beta on 5.3 version. It will be released officially on 6.0 Version. However it is still possible to install the addon and use it in the current version.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
.. warning:: News feature is beta on eXo Platform 5.3. It's planned to be released as generally available in eXo Platform 6.0 Version. It is not pre-packaged in eXo Platform 5.3, however it is still possible to install the addon in order to preview the feature..
```

**Review by plamarque — CHANGES_REQUESTED** [review]
changes suggested

**Review by plamarque — CHANGES_REQUESTED** [review]
IMO you should also add a similar warning on What's new page

**Comment by mzorai** [comment]
> IMO you should also add a similar warning on What's new page

Sorry for dipping in but just as info I didn't add News feature on what's new page I planned to do it when it is released

**Comment by plamarque** [comment]
> > IMO you should also add a similar warning on What's new page
> 
> Sorry for dipping in but just as info I didn't add News feature on what's new page I planned to do it when it is released

 I assumed it was already there. It's fine then.

**Review by plamarque — APPROVED** [review]

**Pull request merged**